project/Build.scala

```diff
@@ -0,0 +1,44 @@
+import sbt._
+import sbt.Keys._
+import sbt.internal.BuildDef
+
+import net.vonbuchholtz.sbt.dependencycheck.DependencyCheckPlugin.autoImport.dependencyCheckFailBuildOnCVSS
+
+object Build extends BuildDef with BuildCommon {
+  override def projects: Seq[Project] = mainProjects ++ testProjects
+  lazy val mainProjects: Seq[Project] = schemaVersions.map(nitfProject)
+  lazy val testProjects: Seq[Project] = schemaVersions.map(testProject)
+
+  private val releaseVersion = 0
+  private val schemaVersions = Seq(/*"3.3",*/ "3.4", "3.5", "3.6")
```

Comment:
Why is 3.3 commented out? Can this be deleted?

Reply:
It shouldn't be. I did a `git reset` and forgot some of the changes. Thanks for noticing!